packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/README.md

```diff
@@ -9,6 +9,17 @@ The Bar Chart Viewer...
 - allows users to view bar chart data (x-axis: categorical values, y-axis:
   numerical values for each category)
 
+## Props
+
+- `barStyles` _(object)_ An object containing properties for the individual bar styles. `padding` is the available configurable style and sets the space between each bar. It defaults to `0.25`.
```

Comment:
I'd specify the shape of the object here. From what you've written, I'd guess that the shape is `{ padding }`?